src/Controller/Exception.php

```diff
@@ -245,11 +246,21 @@ protected function getContextArray(\Exception $exception = null)
             $snippet = false;
         }
 
+        // We might or might not have $this->app['request'] yet, which is used in the
+        // template to show the request variables. Use it, or grab what we can get.
+        if ($this->app['request'] instanceof Request) {
+            $request = $this->app['request'];
```

Comment:
But what if `$app['request_stack']` doesn't exist _yet_? 

I believe that was the issue Carson was encountering, when this is triggered too early in the request cycle.